tests/BrowserKitConfig.php

```diff
@@ -35,4 +35,29 @@ protected function supportsJs()
     {
         return false;
     }
+
+    public function skipMessage($testCase, $test)
+    {
+        if ('Behat\Mink\Tests\Driver\Form\Html5Test' === $testCase && 'testHtml5FormAction' === $test && version_compare($this->getDomCrawlerVersion(), '3.3.0.0-dev', '<')) {
+            return 'Symfony DomCrawler < 3.3 does not support the formAction attribute.';
+        }
+
+        return parent::skipMessage($testCase, $test);
+    }
+
+    private function getDomCrawlerVersion()
+    {
+        $installedFile = __DIR__.'/../vendor/composer/installed.json';
```

Comment:
Looks ugly, but I guess there is no other way to do this unless we have `composer` binary always available to run `composer show -i` and parse the output.